pypit/arwave.py

```diff
@@ -309,6 +309,27 @@ def flexure_obj(slf, det):
     return flex_dict
 
 
+def helio_corr(slf, det, fitsdict):
+
+    from pypit import arvcorr
+    # Calculate
+    helio = arvcorr.calculate(slf, fitsdict, slf._idx_sci[0])
+    hel_corr = np.sqrt( (1. + helio/299792.458) / (1. - helio/299792.458) )
```

Comment:
I'm not so sure about this... you might need to explain to me the origin of the denominator, and why there is a sqrt.

I've thought about this briefly, and I can't think of why the correction should be like this. Naively, I would have thought that the correction (applied on line 328) should be just rearranging the Doppler formula:

```
v/c = (wave_new - wave_old)/wave_old
==>
wave_new = wave_old*(1+v/c)
```

So, I'm clearly missing something.